goa/proj.android-studio/goa/src/main/AndroidManifest.xml

```diff
@@ -21,9 +20,7 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
-    <uses-permission
-        android:name="android.permission.MOUNT_UNMOUNT_FILESYSTEMS"
-        tools:ignore="ProtectedPermissions" />
+    <uses-permission android:name="android.permission.MOUNT_UNMOUNT_FILESYSTEMS" />
```

Comment:
This is a system permission. Remove it since a non-system app will never be able to use these permissions.

Reply:
Will raise a separate PR for permissions alone